synapse_auto_accept_invite/__init__.py

```diff
@@ -86,50 +91,64 @@ async def on_new_event(self, event: EventBase, *args: Any) -> None:
         Args:
             event: The incoming event.
         """
+
         # Check if the event is an invite for a local user.
-        is_invite_for_local_user = (
+        if not (
```

Comment:
This logic was changed to short circuit, mostly to prevent doing redundant work when we already have decided not to accept an invite (e.g. There is no sense in trying to pull the UserInfo from the DB if we're not accepting a group invite).